app/Http/Controllers/User/SettingController.php

```diff
@@ -38,20 +38,28 @@ public function postSaveSetting($phone, $display)
             }
         } else {
             $pattern = '/^(\+84|0)\d{9,10}$/';
-            if (!preg_match($pattern, $phone)) {
-                return response()->json([
-                    'data' => '1',
-                ]);
-            } else {
+            if (preg_match($pattern, $phone)) {
                 $id = Auth::id();
                 $data = [
                     'phone' => $phone,
                 ];
                 $this->user->update($id, $data);
                 $this->usermeta->updateDisplayPhone($id, $display);
+                $dataPhone = [
+                    'key' => 'display_phone',
+                    'user_id' => $id,
+                ];
+                $displayPhone = $this->usermeta->getDataSetting($dataPhone);
+                $data = [
+                    'id' =>$id,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/SettingController.php</br>**Line**: 54</br>Expected 1 space between double arrow and "$id"; 0 found